source/controlTypes/isCurrent.py

```diff
@@ -0,0 +1,52 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2007-2021 NV Access Limited, Babbage B.V.
+
+from typing import Dict
+
+from utils.displayString import DisplayStringStrEnum
+
+
+class IsCurrent(DisplayStringStrEnum):
+	"""Values to use within NVDA to denote 'current' values.
+	These describe if an item is the current item within a particular kind of selection.
+	EG aria-current
+	"""
+	NO = "false"
+	YES = "true"
+	PAGE = "page"
+	STEP = "step"
+	LOCATION = "location"
+	DATE = "date"
+	TIME = "time"
+
+	@property
+	def _displayStringLabels(self):
+		return _isCurrentLabels
+
+	@property
+	def displayString(self):
+		try:
+			return super().displayString
+		except KeyError:
+			return self.YES.displayString

```

Comment:
The previous version logged in this case: `log.debugWarning(f"No translation mapping for: {self}")`

Reply:
This should happen in the `super()` call